build/static/js/2.ea681057.chunk.js.LICENSE.txt

```diff
@@ -4,27 +4,12 @@ object-assign
 @license MIT
 */
 
-/* flatpickr v4.6.1, @license MIT */
-
 /*!
   Copyright (c) 2017 Jed Watson.
   Licensed under the MIT License (MIT), see
   http://jedwatson.github.io/classnames
 */
 
-/*! *****************************************************************************
```

Comment:
Interesting copyrights changes in bundle files, for other patterns I noticed it's putting in a new kind of copyright statement but in this one its just removing em.